Dockerfile

```diff
@@ -173,6 +150,7 @@ COPY --from=downloader /opt/tini /usr/bin/tini
 COPY --from=clboss /usr/local/bin/clboss /usr/local/libexec/c-lightning/plugins/clboss
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl \
```

Comment:
Usually I leave things out that are not used by the container. Sometimes I do add convenience packages (like `vim`) so I have a faster testing cycle. Fine to leave, just providing context.